backend/index.js

```diff
@@ -1,35 +1,48 @@
-import fs from "fs";
-import express from "express";
-
-const MY_COUNTER_VISITED_TIMES_FILE_PATH = 'temp/my-counter-visited-times.txt'
+import fs from 'fs';
+import express from 'express';
+import { VISIT_COUNTER_FILE_PATH } from './constants.js'
 
 const app = express();
 
 let pageVisitedTimesSession = 0;
-let pageVisitedTimesTotal = 0;
+
+let jsonObjForCounter = {
+  pageVisitedTimesTotal: 0,
+  pageVisitedTimesMaxDurSession: 0,
+};
 
 try {
-  if (fs.existsSync(MY_COUNTER_VISITED_TIMES_FILE_PATH)) {
-    pageVisitedTimesTotal = Number(fs.readFileSync(MY_COUNTER_VISITED_TIMES_FILE_PATH, "utf8"))
+  if (fs.existsSync(VISIT_COUNTER_FILE_PATH)) {
+    jsonObjForCounter = JSON.parse(fs.readFileSync(VISIT_COUNTER_FILE_PATH, "utf8"))
```

Comment:
```suggestion
    jsonObjForCounter = JSON.parse(fs.readFileSync(VISIT_COUNTER_FILE_PATH, 'utf8'))
```
I thought this should be catched by prettier 😕